src/sql/workbench/services/connection/browser/connectionBrowseTab.ts

```diff
@@ -105,18 +191,15 @@ export class ConnectionBrowserView extends Disposable implements IPanelView {
 
 	async refresh(items?: ITreeItem[]): Promise<void> {
 		if (this.tree) {
-			if (items) {
-				for (const item of items) {
-					await this.tree.updateChildren({ element: item });
-				}
-			} else {
-				return this.tree.updateChildren();
-			}
+			return this.tree.updateChildren();
```

Comment:
Do we refresh all elements every time now? Is there no way to refresh a particular node?

Reply:
good catch, I should have added comment here, this is not final, I updated this to fix an issue that I can't refresh a particular node, I have an item in the master tracking issue. I am trying to have the PR in a reasonably small size so that it is easier to review. this will be fixed in a later PR